shared/fetcherAnonymous.ts

```diff
@@ -0,0 +1,65 @@
+import axios from "axios";
+import Cookies from "js-cookie";
+
+import getConfig from "next/config";
+import { ITokenEnum } from "@shared/enum";
+
+const { publicRuntimeConfig } = getConfig();
+
+const baseURL = publicRuntimeConfig.BASE_URL;
+
+const api = axios.create({
+  baseURL,
+  timeout: 10000,
+  headers: { "content-type": "application/json"},
+});
+api.interceptors.request.use(
+  async (config) => {
+    const token = Cookies.get(ITokenEnum.anonymousToken);
+    if (token) {
+      config.headers.Authorization = `Bearer ${token}`;
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Consider adding an else clause to handle the case where the token is not found. This will help in debugging if the token is missing for some reason.
```diff
  const token = Cookies.get(ITokenEnum.anonymousToken);
  if (token) {
    config.headers.Authorization = `Bearer ${token}`;
+ } else {
+   console.error('Anonymous token not found');
  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->